src/OpenTelemetry.php

```diff
@@ -0,0 +1,119 @@
+<?php
+/**
+ * Elastic Transport
+ *
+ * @link      https://github.com/elastic/elastic-transport-php
+ * @copyright Copyright (c) Elasticsearch B.V (https://www.elastic.co)
+ * @license   https://opensource.org/licenses/MIT MIT License
+ *
+ * Licensed to Elasticsearch B.V under one or more agreements.
+ * Elasticsearch B.V licenses this file to you under the MIT License.
+ * See the LICENSE file in the project root for more information.
+ */
+declare(strict_types=1);
+
+namespace Elastic\Transport;
+
+use Elastic\Transport\Exception\InvalidArgumentException;
+use Elastic\Transport\Serializer\JsonSerializer;
+use OpenTelemetry\API\Trace\TracerInterface;
+use OpenTelemetry\API\Trace\TracerProviderInterface;
+
+class OpenTelemetry
+{
+    const OTEL_TRACER_NAME = 'elasticsearch-api';
+    // Valid values for the enabled config are 'true' and 'false'
+    const ENV_VARIABLE_ENABLED = 'OTEL_PHP_INSTRUMENTATION_ELASTICSEARCH_ENABLED';
+    /**
+     * Describes how to handle search queries in the request body when assigned to
+     * span attribute.
+     * Valid values are 'raw', 'omit', 'sanitize'. Default is 'omit'
+     */
+    const ALLOWED_BODY_STRATEGIES = ['raw', 'omit', 'sanitize'];
+    const ENV_VARIABLE_BODY_STRATEGY = 'OTEL_PHP_INSTRUMENTATION_ELASTICSEARCH_CAPTURE_SEARCH_QUERY';
+    const DEFAULT_BODY_STRATEGY = 'omit';
+    /**
+     * A string list of keys whose values are redacted. This is only relevant if the body strategy is
+     * 'sanitize'. For example, a config 'sensitive-key,other-key' will redact the values at
+     * 'sensitive-key' and 'other-key' in addition to the default keys
+     */
+    const ENV_VARIABLE_BODY_SANITIZE_KEYS = 'OTEL_PHP_INSTRUMENTATION_ELASTICSEARCH_SEARCH_QUERY_SANITIZE_KEYS';
+
+    const DEFAULT_SANITIZER_KEY_PATTERNS = [
+        'password',
+        'passwd',
+        'pwd',
+        'secret',
+        'key',
+        'token',
+        'session',
+        'credit',
+        'card',
+        'auth',
+        'set-cookie',
+        'email',
+        'tel',
+        'phone'
```

Comment:
These (`email`, `tel`, `phone`) are not present in the Ruby version. Are these standard or did you add them? Maybe I should add them too for consistency across our clients.